Sources_v3/Workers/Background/NewUserQueryUpdater_Tests.swift

```diff
@@ -57,11 +57,29 @@ class NewUserQueryUpdater_Tests: StressTestCase {
         
         // Assert `update(userListQuery` called for each query in DB
         AssertAsync.willBeEqual(
-            env!.userQueryUpdater?.update_queries.map(\.filter.filterHash).sorted(),
+            env!.userQueryUpdater?.update_queries.compactMap(\.filter?.filterHash).sorted(),
             [filter1, filter2].map(\.filterHash).sorted()
         )
     }
     
+    func test_update_isNotCalled_forNilFilterQuery() throws {
+        try database.createUserListQuery(filter: nil)
+        try database.createUser()
+        
+        // Assert `update(userListQuery` is not called for .none filter query
+        AssertAsync.willBeTrue(env!.userQueryUpdater?.update_queries.isEmpty)
+    }
+    
+    func test_newUser_updated_withNilFilterQuery() throws {
+        let id: UserId = .unique
+        
+        try database.createUserListQuery(filter: .none)
```

Comment:
This `.none` is `Optional.none` not `Filter.none` 🤯 It took me a while to understand. However, I'd rather use `nil` :D :D

Reply:
Haha, I changed it to `nil` in another test, but forgot here cause it doesn't throw compilation error 😅